### DISCUSSION TITLE
Tracking accepted and rejected entries

### DISCUSSION THREAD

**Discussion by Nimmo** [Flexget/Flexget#3340] [Q&A] [answered]
I've been trying to experiment with the digest and from_digest plugins to keep a log of all my accepted and rejected entries to then somehow send them somewhere. I'm liking the idea of dumping the report to a file somewhere, but would be interested in seeing where the from_digest can be told to send them to consider other options.

I think my digest looks right:
```accept_reject_digest:
    digest:
      list: rejected_list
      state:
        - rejected
      list: accepted_list
      state:
        - accepted
```

Where I'm struggling is then figuring out what to do with them once I've got them in that digest. Would anyone be able to offer some guidance on how I could create a rejected.txt/accepted.txt file that I could dump somewhere, or even send a daily report to a pushbullet channel?

If there's a better/easier way to do this I'd also be very interested in hearing it.

Thanks,
Nimmo

**Comment by lamski** [comment, marked as answer]
I tried using the `digest` and `email` plugins before with this configuration:
```
  daily_email:
    interval: 1 day
    from_digest:
      list: daily email
    seen: local
    accept_all: yes
    notify:
      entries:
        template: task/html
        what:
          - accepted
          - rejected
          - failed
        via:
          - email:
              to: '{? email.to ?}'
              smtp_host: smtp.gmail.com
              smtp_port: 587
              smtp_username: '{? email.user ?}'
              smtp_password: '{? email.password ?}'
              smtp_tls: yes
      task:
        template: html
        via:
          - email:
              to: '{? email.to ?}'
              smtp_host: smtp.gmail.com
              smtp_port: 587
              smtp_username: '{? email.user ?}'
              smtp_password: '{? email.password ?}'
              smtp_tls: yes
    manual: yes
```

Then in one of my tasks, I added the `digest` config:
```
  task:
    ...
    digest:
      list: daily email
      state:
        - accepted
        - failed
```
You can use a different notification plugin besides `email` but I haven't tried them.
I eventually removed the notification because it was not very useful to me at the time.

**Reply by Nimmo** [reply]
That's great, thanks for that, I've taken it, given it a bash and am hoping that it'll do the job for me.
